web_responsive/static/src/legacy/xml/form_buttons.xml

```diff
@@ -36,9 +44,8 @@
                 data-hotkey="c"
                 t-on-click.stop="create"
             >
-                <t t-call="web_responsive.icon_button">
+                <t t-call="web_responsive.icon_button_create">
                     <t t-set="icon" t-value="'plus'" />
```

Comment:
This can go directly into the template no? I don't see when someone would want a different icon for the same action.